### PR TITLE
Think about expanding interop section

### PR DESCRIPTION
**Pull request by seberg**
This is mainly to start discussion and for Ralf on how we may expand this section.

### CODE REVIEW

File: body.tex
Comment (seberg) [inline body.tex:238]: Honestly, I just realized I have to check that this tool chain is actually has `__array_function__` as a requirement... I assume this is the case though.

Comment (seberg) [reply body.tex:238]: Ah, OK, it is spelled out on this page: https://docs.dask.org/en/latest/gpu.html

EDIT: The video cited embedded there could be a citation: https://on-demand-gtc.gputechconf.com/gtcnew/sessionview.php?sessionName=s9198-dask+and+v100s+for+fast%2c+distributed+batch+scoring+of+computer+vision+workloads

**Comment by jarrodmillman** [comment]
We will continue revising this text and this is in the right direction.  So, I will merge later tonight unless you say otherwise.

File: body.tex
Comment (stefanv) [inline body.tex:237]:
```suggestion
of scaling their computation to the GPU using dask's CuPy integration with
```

File: body.tex
Comment (stefanv) [inline body.tex:240]: Could you explain a bit what "compatibility concerns" means?

File: body.tex
Comment (stefanv) [inline body.tex]:
```suggestion
Thus, we are actively addressing these concerns by revising and extending
```

File: body.tex
Comment (stefanv) [inline body.tex]:
```suggestion
these protocols with the aim of wide
```

File: body.tex
Comment (stefanv) [inline body.tex]:
```suggestion
interoperability that spans the whole scientific Python ecosystem up to and including domain libraries.
```

File: body.tex
Comment (stefanv) [inline body.tex:258]:
```suggestion
The widespread adoption of rich, user defined data types can allow more
```

File: body.tex
Comment (stefanv) [inline body.tex:261]: maybe a bit of clarity on what "enable and interact naturally with different libraries" means.

**Review by stefanv — APPROVED** [review]
@jarrodmillman Agreed that we can merge and keep editing; I suggested some minor tweaks, but can just add those in later when we work more on this section.

**Comment by jarrodmillman** [comment]
I'll let you respond to Stefan's comments before merging.

File: body.tex
Comment (seberg) [reply body.tex:261]: Yeah, I would think of 1. I expect multiple user dtypes can interact (naturally); 2. libraries such as sklearn (or dask) might "just work" with e.g. physical Units (and Categoricals, which they probably really would like to use e.g. for labels); 3. Things being inside a dtype rather than subclass is a better user experience.
I agree its phrased much too fuzzy, but can't think of an improvement quickly right now, maybe the solution is just to expand it to a few sentences though.

Comment (jarrodmillman) [reply body.tex:261]: That sounds reasonable and I agree that you should expand this into a few sentences or more.

**Comment by jarrodmillman** [comment]
I went ahead a merged this.  I made some minor changes and added comments to capture the remaining unresolved issues.  Please feel free to open a new PR to revise this further.

**Pull request merged**
